make.sh

```diff
@@ -20,21 +20,24 @@ usage() {
 Usage: $PROGNAME --bundle-id [BUNDLE_ID_PRODUCTION] --bundle-id-staging [BUNDLE_ID_STAGING] --project-name [PROJECT_NAME]
 Set up an iOS app from tuist template.
 Options:
--h, --help                                   display this usage message and exit
--b, --bundle-id [BUNDLE_ID_PRODUCTION]       the production id (i.e. com.example.package)
--s, --bundle-id-staging [BUNDLE_ID_STAGING]  the staging id (i.e. com.example.package.staging)
--n, --project-name [PROJECT_NAME]            the project name (i.e. MyApp)
+-h, --help                                            display this usage message and exit
+-b, --bundle-id [BUNDLE_ID_PRODUCTION]                the production id (i.e. com.example.package)
+-s, --bundle-id-staging [BUNDLE_ID_STAGING]           the staging id (i.e. com.example.package.staging)
+-n, --project-name [PROJECT_NAME]                     the project name (i.e. MyApp)
+-c, --compile-time-warning [COMPILE_TIME_WARNING]     the compile time warning (i.e. 200,300)
```

Comment:
I think we should add units for clarity, this will be hard to remember in the long run.
```suggestion
-c, --compile-time-warning [COMPILE_TIME_WARNING]     the compile time warning in milliseconds (i.e. 200,300)
```